line/line.go

```diff
@@ -1,14 +1,28 @@
 package line
 
-import config "github.com/tetris-CLI/config"
+import (
+	c "github.com/tetris-CLI/cell"
+	config "github.com/tetris-CLI/config"
+)
 
-//Line 各ブロックにミノが存在しているかどうかを表すStageのライン
-type Line [config.StageWidth]bool
+//Line 複数のMinoによって構成される横一列のライン
```

Comment:
MinoではなくCellでしょうか．Cellを埋めるものがMinoという認識でいます．

Reply:
ここは `Cell` でしたね、修正します！

```suggestion
//Line 複数のCellによって構成される横一列のライン
```